setup.py

```diff
@@ -7,6 +7,9 @@
 
 extras_require = {
     'tester': [
+        # TODO - remove this eth-keys pinning once eth-tester >0.4.0-beta.1 is released
+        # and py-evm v0.3.0-alpha.15 is released
+        "eth-keys>=0.2.1,<0.3",
```

Comment:
Eth-tester v0.4.0-beta.2 looks like it has what we need here (and it's released).
So with a change to the line below, it looks like we can drop this eth-keys version constraint ^.